samples/subsys/zbus/benchmark/src/benchmark.c

```diff
@@ -11,8 +11,19 @@
 #include <zephyr/logging/log.h>
 #include <zephyr/sys/util_macro.h>
 #include <zephyr/zbus/zbus.h>
+
+#if defined(CONFIG_ARCH_POSIX)
+#include "native_rtc.h"
+#define GET_ARCH_TIME_NS() (native_rtc_gettime_us(RTC_CLOCK_PSEUDOHOSTREALTIME) * NSEC_PER_USEC)
```

Comment:
@cfriedt Could you please give us your opinion about line 17? Is this a good practice for `native_posix`?